src/carrier/elacp.fbs

```diff
@@ -43,6 +43,11 @@ table friendmsg {
     msg    : [ubyte];
 }
 
+table friendlmsg {
+    totalsz: uint32;
+    seg    : [ubyte];
+}
+
```

Comment:
table friendbigmsg {  // name it as bigmsg , "lmsg" not self described.
     ext: string;
     totalsz: uint;
     data: [uint8];
}